### ISSUE TITLE
NotImplementError

### ISSUE THREAD

**Issue by judycpChen** [ChenhongyiYang/QueryDet-PyTorch#93]
I find a error:
NotImplementedError: SPCONV_FILTER_HWIO is deprecated. use SPCONV_SAVED_WEIGHT_LAYOUT instead.

How can I work it out?

**Comment by judycpChen** [comment]
![Uploading 1714799270743.png…]()

**Comment by nooneAB** [comment]
I made the same mistake，It seems that the version of spconv will be lowered.

**Comment by judycpChen** [comment]
I have worked it out. I Comments this code in constants.py .
![1714820945069](https://github.com/ChenhongyiYang/QueryDet-PyTorch/assets/151814406/29318745-44dd-44bf-97a7-58b0a4ffd9a9)

**Comment by nooneAB** [comment]
Hello, do you have another problem with inconsistent tensor sizes

**Comment by judycpChen** [comment]
> Hello, do you have another problem with inconsistent tensor sizes

Yes,the problem is "RuntimeError: The expanded size of the tensor must match the existing size at non-singleton dimension,target dimension is [256,3,3,256],but tensor dimension is [3,3,256,256]"

**Comment by nooneAB** [comment]
I changed this problem for a long time yesterday, and found that it could not be solved. I inferred that it was not the comment problem, but the spconv version problem. I will try it again today

**Comment by nooneAB** [comment]
> > 您好，您是否还有其他张量大小不一致的问题
> 
> 是的，问题是“RuntimeError：张量的扩展大小必须与非单例维度的现有大小匹配，目标维度为 [256,3,3,256]，但张量维度为 [3,3,256,256]”

我解决了这个问题，我把2.3.1.0的顺序改成了0.2.3.1后就没有错误了，但是我跑的结果很差，coco的结果只有0.几
